src/SFA.DAS.AdminService.Web.Tests/Controllers/Roatp/RoatpFinancialControllerTests.cs

```diff
@@ -27,22 +30,40 @@ public class RoatpFinancialControllerTests
         private RoatpFinancialController _controller;
         private readonly Guid _applicationId = Guid.NewGuid();
         private string _emailAddress = "Test@test.com";
-
+        protected Mock<IHttpContextAccessor> MockHttpContextAccessor;
+        private FinancialReviewDetails _financialReviewDetails;
+       
         [SetUp]
         public void Before_each_test()
         {
             _roatpOrganisationApiClient = new Mock<IRoatpOrganisationApiClient>();
             _applicationApplyApiClient = new Mock<IRoatpApplicationApiClient>();
             _qnaApiClient = new Mock<IQnaApiClient>();
 
+           
+
+            _financialReviewDetails = new FinancialReviewDetails();
+            MockHttpContextAccessor = SetupMockedHttpContextAccessor();
+
             _controller = new RoatpFinancialController(_roatpOrganisationApiClient.Object,
                 _applicationApplyApiClient.Object,
                 _qnaApiClient.Object,
-                Mock.Of<IHttpContextAccessor>());
+                MockHttpContextAccessor.Object)
+            {
+                ControllerContext = new ControllerContext {HttpContext = new DefaultHttpContext()}
```

Comment:
There's a helper method in `SFA.DAS.AdminService.Common.Testing.MockedObjects` for this